src/auth/user.ts

```diff
@@ -34,3 +34,32 @@ export interface User extends ArenaObject<UserProps> {
   prefs?: UserPrefs
   status: UserStatus
 }
+
+
+// Methods
+
+const getAuthGroups = (user:User):AuthGroup[] => user?.authGroups || []
+
+const isSystemAdmin =  (user:User):boolean => (getAuthGroups(user) || []).some(AuthGroupMethods.isSystemAdminGroup)
+
+const getAuthGroupBySurveyUuid= (surveyUuid: string | undefined, includeSystemAdmin = true) => (user: User):AuthGroup | undefined => {
```

Comment:
do we really need AuthGroup | undefined ?

Reply:
<img width="592" alt="Captura de pantalla 2021-03-11 a las 8 19 03" src="https://user-images.githubusercontent.com/11272068/110750239-78bc7080-8242-11eb-9dc1-ba6996a75f93.png">

Reply:
can't you return null at the end?

Reply:
```
if(something){
return ...
}

return null
```

Reply:
solved by checking `.find` functions signature and finding out it returns `<T> | undefined`
=> `AuthGroup | undefined`